h/browser/chrome/lib/hypothesis-chrome-extension.js

```diff
@@ -137,6 +137,10 @@
         browserAction.deactivate(tabId);
       }
 
+      h.Settings.then(function(settings) {
```

Comment:
A capital letter usually implies a constructor in JavaScript land. I'd prefer it if this were `h.settings`.